vision/objectdetection/color_detector.go

```diff
@@ -10,8 +10,8 @@ import (
 
 // ColorDetectorConfig specifies the fields necessary for creating a color detector.
 type ColorDetectorConfig struct {
-	SegmentSize       int     `json:"segment_size"`
-	Tolerance         float64 `json:"tolerance"`
+	SegmentSize       int     `json:"segment_size"` // TODO(RSDK-586): unit?
+	Tolerance         float64 `json:"tolerance"`    // TODO(RSDK-586): unit?
```

Comment:
SegmentSize is in pixels
Tolerance is a number between 0 and 1 (percetange then?)